README.md

```diff
@@ -8,6 +8,39 @@ First, obtain an App ID and an API key from https://docs.traveltime.com/api/over
 
 You will need to email sales@traveltime.com or book a demo at https://traveltime.com/book-demo to gain access to TimeFilterFast(Proto).
 
+### Testing Each Endpoint Locally
+
+Install [K6](https://k6.io/docs/get-started/installation/)
+
+#### time-map
+
+```bash
+k6 run 
+    -e APP_ID={APP_ID}
+    -e API_KEY={API_KEY}
+    -e HOST=api-dev.traveltimeapp.com //optional
+    -e COUNTRY=gb //optional
+    -e TRANSPORTATION='driving+ferry' //optional
+    -e TRAVEL_TIME=7200 //optional
+    scripts/time-map.js
+```
+
+#### time-filter
+
+```bash
+k6 run 
+    -e APP_ID={APP_ID}
+    -e API_KEY={API_KEY}
+    -e HOST=api-dev.traveltimeapp.com //optional
+    -e COUNTRY=gb //optional
+    -e TRANSPORTATION='driving+ferry' //optional
+    -e TRAVEL_TIME=7200 //optional
+    -e SCENARIO_DESTINATIONS=2000 //optional
```

Comment:
it's not correct. DESTINATIONS are split in scenarios. And each scenario has its own destination. It means that for DESTINATIONS="100,150,200" . The first scenario will use SCENARIO_DESTINATIONS=100, the second SCENARIO_DESTINATIONS=150 and etc.